README.md

```diff
@@ -8,29 +9,119 @@
 
 RIGImageGallery is an image gallery for iOS written in Swift.
 
-The goal is to offer sensible defaults that takes care of most of building an image gallery automatically, and offer easy, block-based customization.
+This library is part of the Raizlabs Interface Guidelines, which are  UI components that offer sensible defaults to help a project get off the ground quickly with components that feel native to the platform, and with easy to use customization options.
+
+RIGImageGallery is a photo gallery meant to provide most of the functionality of the image gallery in the system Photos app, and handle asynchronous loading of images.
```

Comment:
#nth I'd swap this paragraph with the previous one. Say what it is, and then mention RIG.